### ISSUE TITLE
Crash on debian-based linux

### ISSUE THREAD

**Issue by happycamper31** [Librelancer/Librelancer#158]
Hello,
Today I tested Librelancer (this release: librelancer-2023.06.1-ubuntu-amd64.tar.gz). So far I can see the menu just fine & navigate multiplayer & options. When I start a new game I get the crash after the initial cutscene.
I looked into some closed issues and found #87 which sounded similar to me at first .But that seems to have to do with camera settings, while here I can see no hint at that.
Also btw, I wasn't able to figure out which version of Freelancer should be used for Librelancer. So I tested this with multiple versions (1.0, 1.1, 1.25), but all gave me the same crash.
Here the log output:

```
Librelancer has crashed. See the log for more information.

One or more errors occurred. (Object reference not set to an instance of an object.)
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at LibreLancer.Client.CGameSession.HandlePacket(IPacket pkt)
   at LibreLancer.Client.CGameSession.UpdatePackets()
   at LibreLancer.SpaceGameplay.Update(Double delta)
   at LibreLancer.FreelancerGame.Update(Double elapsed)
   at LibreLancer.Game.Run()
   at lancer.MainClass.<>c__DisplayClass0_0.<Main>b__0()
   at LibreLancer.AppHandler.Run(Action action, Action onCrash)
Inner Exception #1:
Object reference not set to an instance of an object.
   at LibreLancer.RoomGameplay.SwitchToRoom(Boolean dolanding)
   at LibreLancer.RoomGameplay..ctor(FreelancerGame g, CGameSession session, String newBase, BaseRoom room, String virtualRoom)
   at LibreLancer.Client.CGameSession.SceneChangeRequired()
   at LibreLancer.Client.CGameSession.LibreLancer.Net.Protocol.IClientPlayer.BaseEnter(String _base, Int32 objective, String[] rtcs, NewsArticle[] news, SoldGood[] goods, NetSoldShip[] ships)
   at LibreLancer.Net.Protocol.GeneratedProtocol.HandleClientPacket(IPacket pkt, IClientPlayer player, INetResponder res)

Librelancer Version: 2023.06.1
```

Maybe not related to this issue: but the game also crashes immediately when I put the master volume to 0.

```
Librelancer has crashed. See the log for more information.

Invalid Value
   at LibreLancer.Media.Al.CheckErrors()
   at LibreLancer.Media.Al.alListenerf(Int32 param, Single value)
   at LibreLancer.Media.AudioManager.set_MasterVolume(Single value)
   at LibreLancer.FreelancerGame.Load()
   at LibreLancer.Game.Run()
   at lancer.MainClass.<>c__DisplayClass0_0.<Main>b__0()
   at LibreLancer.AppHandler.Run(Action action, Action onCrash)

Librelancer Version: 2023.06.1
```

Hopefully we can figure this out together

**Comment by CallumDev** [comment]
Hi, are you able to try with the latest daily available at https://librelancer.net/downloads.html ? (Use the linux amd64 build)

**Comment by happycamper31** [comment]
Hi Callum,
Thanks for the quick response. I get a little bit farther now! Now the game properly skips the intro cutscene, but then crashes with the following message:

```
Librelancer has crashed. See the log for more information.

Object reference not set to an instance of an object.
   at LibreLancer.Net.ObjNetId.op_Implicit(GameObject obj)
   at LibreLancer.Missions.Act_CallThorn.<>c__DisplayClass3_0.<Invoke>b__0()
   at LibreLancer.Server.Player.UpdateMissionRuntime(Double elapsed)
   at LibreLancer.Server.GameServer.Process(TimeSpan time, TimeSpan totalTime, UInt32 currentTick)
   at LibreLancer.Server.FixedTimestepLoop.Start()
   at LibreLancer.Server.GameServer.GameThread()

Librelancer Version: 4b83f0f-git (20240101)
```
BR happycamper

**Comment by CallumDev** [comment]
These issues should now be fixed in the daily a085e9f (volume and objnetid related)

**Comment by happycamper31** [comment]
Nice, I tested the daily after the fix. Seems to work, at least I tested the start of the game :) 
Thank you!
